modules/lib/src/main/resources/assets/js/app/wizard/ContentWizardPanel.ts

```diff
@@ -2686,6 +2689,14 @@ export class ContentWizardPanel
         return this.splitPanel;
     }
 
+    isTranslatable(): boolean {
+        const content = this.getContent();
+
+        return AI.get().canTranslate() &&
+               (this.isContentExistsInParentProject() && content.hasOriginProject()) &&
+               !!ProjectContext.get().getProject().getLanguage();
+    }

```

Comment:
There's no check here for that the form only contains input types we can translate (TextLine, TextArea & HtmlArea). Or are we going to add this later?

Reply:
Nevermind that - the check is in Saga